bin/bench

```diff
@@ -3,16 +3,32 @@
 const path = require('path');
 const program = require('commander');
 const run = require('../lib/run');
+const mkdirp = require('mkdirp').sync;
+const { dirname, readFileSync, writeFileSync } = require('fs');
```

Comment:
`dirname` is unused and should be removed. (I should set up linting to catch this...)